utils/threshAlgos.py

```diff
@@ -1229,19 +1253,48 @@ def sbitRateAnalysis(chamber_config, rateTree, cutOffRate=0.0, debug=False, outf
                     pass
                 pass
             pass
+
+        # print points in a table
+        if perchannel == False:
+            print(tabulate(inflectTable, headers = ['Geo Addr', 'VFAT Number', 'ARM DAC Inflection Pt'], tablefmt='orgtbl') )
+            inflectTableFile = file("{0}/{1}/inflectionPointTable.txt".format(elogPath,chamber_config[ohKey],scandate), "w")
+            inflectTableFile.write(tabulate(inflectTable, headers = ['Geo Addr', 'VFAT Number', 'ARM DAC Inflection Pt'], tablefmt='orgtbl') )
```

Comment:
It seems inflectTable has information for *all* links (maybe I'm wrong).  But you're writing this file *per* link.  For the output file I would prefer to *not* have this information for all links but only the per link info in the per link file.

Reply:
The commit after this addresses this now and handles some additional error cases. 

https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/231/commits/09da4b11772349a57ef42208f654c5540c742066